src/rawtoaces_util/image_converter.cpp

```diff
@@ -1433,11 +1496,109 @@ bool ImageConverter::configure(
+        }
+        std::cerr << std::endl;
+
+        if ( !settings.illuminant.empty() )
+        {
+            std::cerr << "  Illuminant: " << settings.illuminant << std::endl;
+        }
+
+        if ( !settings.custom_camera_make.empty() ||
+             !settings.custom_camera_model.empty() )
+        {
+            std::cerr << "  Camera override: " << settings.custom_camera_make
+                      << " / " << settings.custom_camera_model << std::endl;
+        }
+
+        if ( settings.WB_method == Settings::WBMethod::Box )
+        {
+            std::cerr << "  WB box: [" << settings.WB_box[0] << ", "
+                      << settings.WB_box[1] << ", " << settings.WB_box[2]
+                      << ", " << settings.WB_box[3] << "]" << std::endl;
+        }
+
+        if ( settings.WB_method == Settings::WBMethod::Custom )
+        {
+            std::cerr << "  Custom WB: [" << settings.custom_WB[0] << ", "
+                      << settings.custom_WB[1] << ", " << settings.custom_WB[2]
+                      << ", " << settings.custom_WB[3] << "]" << std::endl;
+        }
+
+        if ( settings.matrix_method == Settings::MatrixMethod::Custom )
+        {
+            std::cerr << "  Custom matrix:" << std::endl;
+            for ( int i = 0; i < 3; i++ )
+            {
+                std::cerr << "    [" << settings.custom_matrix[i][0] << " "
+                          << settings.custom_matrix[i][1] << " "
+                          << settings.custom_matrix[i][2] << "]" << std::endl;
+            }
+        }
+
+        std::cerr << "  Crop mode: ";
+        switch ( settings.crop_mode )
+        {
+            case Settings::CropMode::Off: std::cerr << "off"; break;
+            case Settings::CropMode::Soft: std::cerr << "soft"; break;
+            case Settings::CropMode::Hard: std::cerr << "hard"; break;
+        }
+        std::cerr << std::endl;
+
+        if ( settings.crop_box[2] > 0 && settings.crop_box[3] > 0 )
+        {
+            std::cerr << "  Crop box: [" << settings.crop_box[0] << ", "
+                      << settings.crop_box[1] << ", " << settings.crop_box[2]
+                      << ", " << settings.crop_box[3] << "]" << std::endl;
+        }
+
+        std::cerr << "  Demosaic: " << settings.demosaic_algorithm << std::endl;
+        std::cerr << "  Headroom: " << settings.headroom << std::endl;
+        std::cerr << "  Scale: " << settings.scale << std::endl;
+        std::cerr << "  Output dir: "
+                  << ( settings.output_dir.empty() ? "<same as input>"
+                                                   : settings.output_dir )
+                  << std::endl;
+        std::cerr << "  Overwrite: " << ( settings.overwrite ? "yes" : "no" )
+                  << std::endl;
+        std::cerr << "  Create dirs: "
+                  << ( settings.create_dirs ? "yes" : "no" ) << std::endl;
+        std::cerr << "  Verbosity: " << settings.verbosity << std::endl;
+    }
+
```

Comment:
I feel line all this above can live in own private function. Takes settings in and prints it out. `print_out_settings()` or something.